client/blocks/product-selector/index.jsx

```diff
@@ -409,8 +451,10 @@ const connectComponent = connect( ( state, { products, siteId } ) => {
 	const availableProducts = getAvailableProductsList( state );
 
 	return {
+		availableProducts,
 		currencyCode: getCurrentUserCurrencyCode( state ),
 		currentPlanSlug: getSitePlanSlug( state, selectedSiteId ),
+		fetchingSitePlans: isRequestingSitePlans( state ),
```

Comment:
Let's add `fetchingSitePlans` to the [connected `propTypes` definition](https://github.com/Automattic/wp-calypso/pull/37396/files#diff-dfff688a37f93f6dc6effc4f8b83730dR430) above.

Reply:
Nice! 👍 Done in https://github.com/Automattic/wp-calypso/pull/37396/commits/a123b5f15c4a6fd0b5321dd995f3328b388678a8.